api/src/controllers/website.js

```diff
@@ -81,6 +81,45 @@ async function deleteWebPage(req, res, next) {
  * @returns 200: changes updated in DB
  * @returns 500: Server error while saving to DB
  */
+async function updatePublicationOptions(req, res, next) { // eslint-disable-line no-unused-vars
+  const updatedPubOptions = req.body;
+  const { teamId: _id } = req.params;// TODO: teamId get from the token instead of parameters?
+  try {
+    await Website.updateOne(
+      { teamId: _id },
+      {
+        $set: {
+          publicationOptions: updatedPubOptions,
+        },
+      },
+    );
+    return res.status(200).json(updatedPubOptions);
+  } catch (err) {
+    return res.send(fillErrorObject(500, 'Server error', [err.errors]));
+  }
+}
+
+async function resetWebPage(req, res, next) { // eslint-disable-line no-unused-vars
```

Comment:
Similar Q, why am endpoint specifically for this?
I’d imagine this to be part of broader delete as well?